### PR TITLE
Remove autopub

### PR DESCRIPTION
**Pull request by lucasefe**
We no longer need to auto publish messages. Sentry's been around for a while. 

### Steps to merge

:+1: of the team

/cc @zendesk/zendesk-radar

### Risks

Low.

### CODE REVIEW

File: core/lib/resources/presence/presence_manager.js [hunk 1]
```diff
@@ -262,15 +240,6 @@ PresenceManager.prototype.processRedisEntry = function(message, callback) {
...
   } else {
-    if (this.isExpired(message)) {
-      logging.info('#autopub - received online message - expired', message,
-                                                                this.scope);
-
-      // Orphan autopub redis entry: silently remove from redis
-      Persistence.deleteHash(this.scope, userId + '.' + socketId);
-      message.explicit = false;
-    }
-
```
Comment (vanchi-zendesk) [inline core/lib/resources/presence/presence_manager.js]: A bit more can be removed. (line 218 - 225).

File: core/lib/resources/presence/presence_manager.js [hunk 2]
```diff
@@ -216,13 +200,7 @@ PresenceManager.prototype._implicitDisconnect = function(socketId, userId,
...
 PresenceManager.prototype.isOnline = function(message) {
   // Return with online, or with sentry, or with unexpired legacy msg
-  return (message.online && (message.sentry ||
-       (message.at && message.at >= (Date.now() - PresenceManager.autoPubTimeout))));
-};
-
-PresenceManager.prototype.isExpired = function(message) {
-  return (message.online && !message.sentry &&
-      message.at && message.at < (Date.now() - PresenceManager.autoPubTimeout));
+  return (message.online && message.sentry);
```
Comment (vanchi-zendesk) [inline core/lib/resources/presence/presence_manager.js]: we can now assume that message.sentry is always present. So this function is meaningless.

Comment (lucasefe) [inline core/lib/resources/presence/presence_manager.js]: Which one? IsOnline, right?

Comment (vanchi-zendesk) [inline core/lib/resources/presence/presence_manager.js]: Ya. isOnline is now simply message.online. So you might as well use message.online directly.

**Comment by vanchi-zendesk** [comment]
:+1: on green.

**Pull request merged**